docs/user/dashboard/lens.asciidoc

```diff
@@ -261,6 +282,16 @@ The following component menus are available:
 
 * *Left axis*, *Bottom axis*, and *Right axis* &mdash; Specify how you want to display the chart axes. For example, add axis labels and change the orientation and bounds.
 
+[float]
+[[open-lens-data-in-discover]]
+=== Open the data in Discover
+
+When your visualization includes one data view, you can open and explore the visualization data in *Discover*.
+
+To get started, click *Open in Discover* in the toolbar.
```

Comment:
We've updated this: `Open in Discover` => `View raw data`

Reply:
There was a discussion that circulated a long time ago that we should not use `Explore raw data` in the visualization editors, and we removed these options. We should change this back to `Open in Discover`.

Reply:
++ to what @KOTungseth said about using the term "raw data".  The two options Kaarina suggests are easier to understand, IMO.

Reply:
of these two I think "Open in Discover"...I like "Explore your data" but that's also what you're doing in the visualizations too so we need some descriptive way of saying "see the underlying data behind this visualization"

It would be nice if we could avoid all of the "Open in _____" but that can be a later phase thing for us to figure out soon

Reply:
I'll defer to ya'll for the final wording for this. For the sake of documenting it, the reason I am hesitant to choose "Open in Discover" is because it demands that the user be familiar with the marketing names and use cases of our products (Discover in this case). Newer users may not be familiar with app names, such as Discover, Lens, etc. My preference would be to include some kind of wording that explains the action being taken (similar to how we used "Edit visualization in Lens" for that functionality in TSVB).

Reply:
Is there enough room for `Explore data in Discover`?

Reply:
What do you think?

<img width="1439" alt="Screen Shot 2022-04-06 at 8 53 47 AM" src="https://user-images.githubusercontent.com/315764/161991329-5d38ca54-c73d-4258-8d0f-76e4e01ce83a.png">
<img width="330" alt="Screen Shot 2022-04-06 at 8 55 32 AM" src="https://user-images.githubusercontent.com/315764/161991363-3a172415-93dc-438c-b10e-a7e955ccad71.png">